lib/pharos/scripts/configure-etcd.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -e
+
+if [ ! -e /etc/kubernetes/kubelet.conf ]; then
+  mkdir -p /etc/systemd/system/kubelet.service.d
+  cat <<EOF >/etc/systemd/system/kubelet.service.d/5-pharos-etcd.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/kubelet --pod-manifest-path=/etc/kubernetes/manifests/ --read-only-port=0 --cadvisor-port=0 --address=127.0.0.1
+EOF
+
+  apt-mark unhold kubelet
+  apt-get install -y kubelet=${KUBE_VERSION}-00
+  apt-mark hold kubelet
+fi
+
+mkdir -p /etc/kubernetes/manifests
+cat <<EOF >/etc/kubernetes/manifests/pharos-etcd.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ""
+  labels:
+    component: etcd
+    tier: control-plane
+  name: etcd
+  namespace: kube-system
+spec:
+  containers:
+  - command:
+    - etcd
+    - --name=${PEER_NAME}
+    - --cert-file=/etc/kubernetes/pki/etcd/server.pem
+    - --key-file=/etc/kubernetes/pki/etcd/server-key.pem
+    - --trusted-ca-file=/etc/kubernetes/pki/ca.pem
+    - --peer-trusted-ca-file=/etc/kubernetes/pki/ca.pem
+    - --data-dir=/var/lib/etcd
+    - --peer-cert-file=/etc/kubernetes/pki/etcd/peer.pem
+    - --peer-key-file=/etc/kubernetes/pki/etcd/peer-key.pem
+    - --listen-client-urls=https://${PEER_IP}:2379
```

Comment:
Consider `--listen-client-urls=https://0.0.0.0:2379` as this prevents using the straightforward `https://localhost:2379/`?

Although if the peer IP is the private IP, then that would also expose etcd on the public IP... so this is safer in that case.

Reply:
Or this can take multiple URLs, so `--listen-client-urls=https://localhost:2379,https://${PEER_IP}:2379` would work nicely

Reply:
Fixed.